test/Riok.Mapperly.Tests/Mapping/ObjectPropertyFromSourceTest.cs

```diff
@@ -0,0 +1,120 @@
+using Riok.Mapperly.Diagnostics;
+
+namespace Riok.Mapperly.Tests.Mapping;
+
+public class ObjectPropertyFromSourceTest
+{
+    [Fact]
+    public void WithManualMappedProperty()
```

Comment:
Can you also add a test case for the following scenarios?
* to map to a constructor parameter
* to map to a init only property
* to map to a tuple value
* in a queryable projection mapping (probably in another test class)
* with a nullable object (where the source is nullable)